### ISSUE TITLE
Polymer 0.9 ?

### ISSUE THREAD

**Issue by jmcgdz** [Juicy/juicy-html#9]
I used this in 0.5 and am now upgrading and it doesn't seem to work as it used to.

Is there a known replacement in polymer 0.9 to handle this?

**Comment by tomalec** [comment]
Thanks for feedback :)

We definitely plan to port it to new Polymer. 
As we use it with TemplateBinding and PolymerExpressions we waited for Polymer > 0.5 to get more stable, especially that TemplateBinding was completely replace with new stuff.

We'll start it soon, probably after Google I/O ;)

**Comment by tomalec** [comment]
Polymer Template Binding seems to be still there in Polymer 1.0 
https://www.polymer-project.org/1.0/docs/devguide/templates.html#dom-bind

So we are right, to update it to  1.0 :)

**Comment by tomalec** [comment]
So today they released 1.0.0 followed by 1.0.1 patch, and I'm no longer so enthusiastic.

You can check it on branch [1.0.x](https://github.com/Juicy/juicy-html/tree/1.0.x).
Most of the core features works:
 - inserting inline HTML,
 - loading HTML from external file

However, binding withing inserted HTML does not work, I'm not sure if Polymer 1.0.x is worth trying. Maybe we should wait for 1.1.x, as even not all paper-/iron- elements works with it, and configuring template binding within Polymer element is a huge pain.

Or maybe we should make core `juicy-html` features work without Polymer at all, and do not bother about poor support for data binding in Polymer > 0.5.x

**Comment by egyptianbman** [comment]
First, thanks for this.  It's a life saver!

For the sake of ease of maintenance, putting in the time to be Polymer-agnostic may be worth it IMHO as Polymer is a constantly evolving beast.

If you were to go that route, how long do you think it would take for this repo to be 1.0-ready?

**Comment by tomalec** [comment]
I have just released [1.0.0-pre.1](https://github.com/Juicy/juicy-html/releases/tag/1.0.0-pre.1). Which removes Polymer dependency, so it works fine with Polymer 1.1 :)

I will release 1.0.0, probably next week, once we will make sure that it works fine in all (at least ours ;) ) use cases.

If you use Polymer migration will require small changes, but you will need to change your `<template bind>`s any way to upgrade your code to Polymer 1.0. 
Plus, I think it will make code cleaner, more declarative, flow (stamping, re-flows, etc.) gets simpler, and the performance should get better, as `juicy-html` is now more lightweight, creates less elements, and does not inherit all Polymer magic features.


So, if previously your code looked like:
```html
<template bind id="tpl">
    <template is="juicy-html" content="{{html}}" bind="{{data}}"></template>
</template>
<script>
    document.getElementById("tpl").model = {
        html: "<h1>Hello <strong>{{name}}</strong></h1>",
        data: {
            name: "World"
        }
    };
</script>
```
Now, it should look like
```html
<template is="juicy-html" 
    content='
      <template is="dom-bind">
        <h1>Hello <strong>{{model.name}}</strong></h1>
      </template>'
    model="{
        name: "World"
    }">
</template>
```

Naturally, you can also wrap `<template is="juicy-html">` within `dom-bind` and provide `content`, and `model` attributes using binding. (check our demos, for more examples)
